src/utils/shared.ts

```diff
@@ -17,6 +17,15 @@ export const toHtmlSafeId = (
     .toLowerCase()}${suffix ? suffix : ''}`;
 };
 
+export const validateSearchField = (value?: string) => {
```

Comment:
some relevant discussion on this https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/pull/720#discussion_r1094409885 food for thought for future :grinning: